games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="http://www.chocolate-doom.org"
+COPYRIGHT="Chocolate Doom Team"
```

Comment:
Also, iD software still has copyright on some of the code. Please research the copyright owners.